.env.example

```diff
@@ -230,6 +230,12 @@ AUTH_BEARER_TOKEN_VERCEL=
 E2E_TEST_APPLE_CALENDAR_EMAIL=""
 E2E_TEST_APPLE_CALENDAR_PASSWORD=""
 
+# ALBY
+# Used for the Alby payment app / receiving Alby payments
+# Get it from: https://getalby.com/developer/oauth_clients
+NEXT_PUBLIC_ALBY_CLIENT_ID=
+NEXT_PUBLIC_ALBY_CLIENT_SECRET=
+
```

Comment:
moving this to .env.appStore.example

Locally I don't know why but in my setup I still have to set this on .env